src/Stratis.Bitcoin.Features.SignalR/Events/WalletGeneralInfoClientEvent.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using Stratis.Bitcoin.EventBus;
+using Stratis.Bitcoin.Features.Wallet.Models;
+
+namespace Stratis.Bitcoin.Features.SignalR.Events
+{
+    /// <summary>
+    /// Marker type for Client
+    /// </summary>
+    public class WalletGeneralInfo
```

Comment:
Is this needed?

Reply:
Not needed but made a conscious decision to do this as its the type that the client uses to subscribe to the event, so just looks a bit cleaner. Could put `WalletGeneralInfoClientEvent` as the type in the UI but it then loses consistency with the ISignalsEvents

Reply:
Ah sorry I meant the empty constructor ;)

Reply:
There isn't an empty constructor